### PR TITLE
docs: gitbook and FAQ fixes

### PR DESCRIPTION
**Pull request by nyrahul**
**Purpose of PR?**:

* Gitbook documentation fixes.
* General doc fixes
* FAQ link fixes

**Checklist:**
- [x] Bug fix.
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [x] Commit has unit tests
- [x] Commit has integration tests

### CODE REVIEW

File: getting-started/FAQ.md [hunk 1]
```diff
@@ -2,42 +2,43 @@
...
+* KubeArmor also runs in systemd mode so can directly run and protect Virtual Machines or Bare-metal machines too.
+* Pod Security Context cannot leverage BPF-LSM at all today. BPF-LSM provides more programmatic control over the policy rules.
 * Pod Security Context do not manage abstractions. As an example, you might have two nodes with Ubuntu, two nodes with Bottlerocket. Ubuntu, by default has AppArmor and Bottlerocket has BPF-LSM and SELinux. KubeArmor internally picks the right primitives to use for enforcement and the user do not have to bother explicitly stating what to use.
 </details>
 
-<details><summary><h4>What is visibility that I hear of in KubeArmor and how to get visibility information?</h4></summary>  
+<details><summary><h4>What is visibility that I hear of in KubeArmor and how to get visibility information?</h4></summary>
 
 KubeArmor, apart from been a policy enforcement engine also emits pod/container visibility data. It uses an eBPF-based system monitor which keeps track of process life cycles in containers and even nodes, and converts system metadata to container/node identities. This information can then be used for observability use-cases.
```
Comment (Ankurk99) [inline getting-started/FAQ.md:44]: Can you please change the `karmor log --json` to `karmor logs --json` on line 45?

File: README.md [hunk 2]
```diff
@@ -27,16 +27,16 @@ KubeArmor leverages [Linux security modules \(LSMs\)](https://en.wikipedia.org/w
...
 ### Contributors :busts_in_silhouette:
 
-* :octocat: [Contribution Guide](contribution/contribution_guide.md)
+* :blue_book: [Contribution Guide](contribution/contribution_guide.md)
 * :technologist: [Development Guide](contribution/development_guide.md), [Testing Guide](contribution/testing_guide.md)
-* :raising_hand_woman: [Join KubeArmor Slack](https://join.slack.com/t/kubearmor/shared_invite/zt-1ltmqdbc6-rSHw~LM6MesZZasmP2hAcA)
+* :raised_hand: [Join KubeArmor Slack](https://join.slack.com/t/kubearmor/shared_invite/zt-1ltmqdbc6-rSHw~LM6MesZZasmP2hAcA)
 * :question: [FAQs](getting-started/FAQ.md)
 
 ### Biweekly Meetup
```
Comment (Ankurk99) [inline README.md]:
```suggestion
### Biweekly Meeting
```
Will improve clomonitor score.

**Review by Ankurk99 — APPROVED** [review]
LGTM, just a few comments - not related to the new changes but will be good to address.

File: getting-started/FAQ.md (hunk 1 above)
Comment (nyrahul) [reply getting-started/FAQ.md:44]: Great catch. Fixed

**Pull request merged**